build.gradle

```diff
@@ -69,9 +69,9 @@ subprojects { project ->
   version = VERSION_NAME
 
   repositories {
+    google()
```

Comment:
This should come after `mavenCentral()`

Reply:
Thanks for pointing out, I updated it. 👍